package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "voxa-ama",
+  "version": "0.2.4",
+  "description": "Integrate Amazon Mobile Analytics analytics into your Alexa apps using the voxa framework",
+  "main": "index.js",
+  "scripts": {
+    "test": "istanbul cover _mocha -- --recursive tests",
+    "test-ci": "JUNIT_REPORT_PATH=junit.xml istanbul cover _mocha -- --recursive --colors --reporter mocha-jenkins-reporter tests",
+    "cobertura": "istanbul report cobertura --root coverage",
+    "lint": "eslint lib test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mediarain/voxa-ama.git"
+  },
+  "keywords": [
+    "alexa",
+    "skills",
+    "apps",
+    "framework",
+    "echo",
+    "voxa",
+    "analytics",
+    "ama",
+    "amazon",
+    "mobile",
+    "analytics",
+    "amazon mobile analytics"
+  ],
+  "author": "Rain Agency <npm@rain.agency> (http://rain.agency)",
+  "engines": {
+    "node": ">=4.3.2 <=6.10"
```

Comment:
let's stop restricting the maximum node version required, if they want to host voxa on an ec2 instance with node 8 it's their deal